Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -309,4 +312,244 @@ theorem psum_isSymmetric (n : ℕ) : IsSymmetric (psum σ R n) := rename_psum _
 
 end PowerSum
 
+section Exponent
+
+open Classical
```

Comment:
Is this really necessary? It's often a better idea to avoid it and fix the proofs that need `Classical`.